### DISCUSSION TITLE
Comparision with Apache TVM

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [beehive-lab/TornadoVM#497] [Q&A] [answered]
Good day.
May I ask how TorndoVM is different from Apache TVM? https://tvm.apache.org/

**Comment by mikepapadim** [comment, marked as answer]
Hello, 

Apache TVM purely focuses on ML/DL workloads and not intend to be used for general purpose workloads or data-analytics.
It provides intergration with TensorFlow, Keras, PyTorch, ONNXRT etc. 
The front-end is written in python, while the compile and runtime are C++ with FFI binding to python. 
Also, for intergration with languages running on the JVM, it provides some minimal wrappers for Java.

**Reply by andrii0lomakin** [reply]
@mikepapadim thank you.
